src/ParallelEnv.f90

```diff
@@ -200,6 +201,16 @@ SUBROUTINE ParEnvType_clear_absintfc(myPE)
         gather_SLK0_MPI_Env_type, &
         gather_SLK1_MPI_Env_type, &
         gather_SIK1_MPI_Env_type
+
+    !> @copybrief ParallelEnv::gatherV_str1D_MPI_ENV_type
+    !> @copydetails ParallelEnv::gatherV_str1D_MPI_ENV_type
+    PROCEDURE,PASS,PRIVATE :: gatherV_str1D_MPI_ENV_type
+    !> @copybrief ParallelEnv::gatherV_str2D_MPI_ENV_type
+    !> @copydetails ParallelEnv::gatherV_str2D_MPI_ENV_type
+    PROCEDURE,PASS,PRIVATE :: gatherV_str2D_MPI_ENV_type
+    !>
+    GENERIC :: gatherV => gatherV_str1D_MPI_ENV_type,gatherV_str2D_MPI_ENV_type
```

Comment:
Couldn't get this under the `gather` generic?

Reply:
I started with the same generic, but hit a snag (probably of my own making) with the arguments in the interface. Not sure which way around gather/gatherv makes more sense in the actual name scheme.

Reply:
Hmm... ok... do you remember the error?  If not, could you try it again and let me know what it was?  I've run into errors with `GENERIC` when we used deferred shape arrays (`REAL(SRK),DIMENSION(*) :: ...`), but I feel like the current `gather` procedures should work fine.